### DISCUSSION TITLE
Does Coolify Have Rollback?

### DISCUSSION THREAD

**Discussion by darrylmorley** [coollabsio/coolify#1667] [Q&A]
Hi,

Just taking a look at Coolify, but wondering if Coolify provides a rollback feature like Caprover?

Thanks

**Comment by katmai7** [comment]
Check this https://coolify.io/docs/applications#rollbacks

**Reply by okunamayanad** [reply]
That link didn't age well..

[here's the new link](https://coolify.io/docs/knowledge-base/applications#rollbacks)

**Comment by AlejandroAkbal** [comment]
Yes, please close this issue
